bbc/simorgh
# Lite ATI Analytics script

Resolves https://jira.dev.bbc.co.uk/browse/WSTEAM1-1420

Overall changes
======
- Adds a dedicated script for sending ATI page view for the Lite site instead of using the tracking pixel, taking inspiration for the recently added Opera Mini extreme script
- Retains support for when JavaScript is disabled by using the `noscript` tracking pixel in that instance

Testing
======
1. Visit http://localhost:7080/pidgin.lite?renderer_env=live
2. Confirm that the ATI page view beacon is sent in the 'Network > Fetch/XHR' tab. E.g: <img width="1332" alt="Screenshot 2024-11-04 at 10 05 51" src="https://github.com/user-attachments/assets/55e9ce30-bb4d-4287-9e1c-470f7f361bf3">
3. Select the 'Img' tab and refresh the page
4. Confirm that no ATI tracking pixel events are sent
5. Repeat steps for http://localhost:7080/pidgin/articles/cd14lllvdveo.lite?renderer_env=live 
6. Disable JavaScript
7. Refresh the pages and confirm that the ATI page view beacon **does not appear in the 'Fetch/XHR'** tab. It should now appear in the **'Img'** tab as it will be using the `noscript` tracking pixel

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
